webview/src/experiments/components/table/header/TableHeaderCell.tsx

```diff
@@ -51,16 +45,11 @@ const getHeaderPropsArgs = (
         [styles.paramHeaderCell]: columnWithGroup.group === ColumnType.PARAMS,
         [styles.metricHeaderCell]: columnWithGroup.group === ColumnType.METRICS,
         [styles.depHeaderCell]: columnWithGroup.group === ColumnType.DEPS,
-        [styles.createdHeaderCell]: header.id === 'Created',
-        [styles.firstLevelHeader]: isFirstLevelHeader(header.column.id),
-        [styles.leafHeader]: header.subHeaders === undefined,
-        [styles.menuEnabled]: sortEnabled,
-        [styles.sortingHeaderCellAsc]: sortOrder === SortOrder.ASCENDING,
-        [styles.sortingHeaderCellDesc]:
-          sortOrder === SortOrder.DESCENDING && !header.isPlaceholder,
-        [styles.oneRowHeaderCell]: onlyOneLine
```

Comment:
`menuEnabled`, `sortingHeaderCellAsc`, and `sortingHeaderCellDesc` weren't used in the styles file so I deleted the classes.